test/client/fixtures/sandbox/node/window-test.js

```diff
@@ -76,18 +76,82 @@ test('parameters passed to the native function in its original form', function (
         checkNativeFunctionArgs('setTimeout', 'setTimeout', window);
     }
 
-    if (window.history.pushState && window.history.replaceState) {
-        checkNativeFunctionArgs('pushState', 'historyPushState', window.history);
-        checkNativeFunctionArgs('replaceState', 'historyReplaceState', window.history);
-    }
-
     var documentFragment = document.createDocumentFragment();
 
     checkNativeFunctionArgs('querySelector', 'documentFragmentQuerySelector', documentFragment);
     checkNativeFunctionArgs('querySelectorAll', 'documentFragmentQuerySelectorAll', documentFragment);
     checkNativeFunctionArgs('dispatchEvent', 'windowDispatchEvent', window);
 });
 
+if (window.history.replaceState) {
+    asyncTest('replace history state with a various types of url', function () {
```

Comment:
You changed `replaceState` and `pushState` functions.
So, need to add test cases for `pushState` function